db/migrate/20211117174431_add_user_to_task.rb

```diff
@@ -0,0 +1,9 @@
+class AddUserToTask < ActiveRecord::Migration[6.1]
+  def change
+    change_table :tasks do |t|
+      t.integer :user_id, null: true
```

Comment:
Use `reference` https://edgeguides.rubyonrails.org/active_record_migrations.html#references

Reply:
Ok

Reply:
Have you run this migration?